src/lib/executors/Node.ts

```diff
@@ -127,6 +147,7 @@ export default class Node extends Executor<Events, Config, NodePlugins> {
 
 		this.on('coverage', message => {
 			this._coverageMap.merge(message.coverage);
+			message.coverage = mergeCoverageData(duplicate(this._initialCoverage), message.coverage);
```

Comment:
Rather than altering message data for other listeners, maybe we should:

1. do the duplication in `emit`
2. emit the initial coverage with its own 'coverage' event rather than merging new data into it here